### PR TITLE
NEARESTCORNER must be onscreen  (addresses #1294

### PR DESCRIPTION
**Pull request by rmkaplan**
Mouse jumps to the nearest onscreen corner of the ghost region

### CODE REVIEW

**Review by masinter — COMMENTED** [review]
I think there's a problem with these merge commits, because they show no explanation when you go to view the PR on github.
So it's hard to see this from GitHub, maybe PRC shows it?
What I was wondering was: is it possible for a window to be completely off screen? What about if you resize the screen (e.g., log into online.interlisp.org from a phone)?  

Perhaps we need a separate issue for dealing with resizing without logout.

File: lispusers/MODERNIZE [hunk 1]
```diff
@@ -325,21 +325,46 @@
...
+    (LET ((LEFT (FETCH (REGION LEFT) OF REGION))
+          (RIGHT (FETCH (REGION RIGHT) OF REGION))
+          (TOP (FETCH (REGION TOP) OF REGION))
+          (BOTTOM (FETCH (REGION BOTTOM) OF REGION))
+          X Y)
+
+         (* ;; "If the nearest corner is offscreen, pick the other one.")
+
+         (SETQ X (if (OR (ILESSP LEFT 0)
+                         (IGREATERP LEFT SCREENWIDTH))
```
Comment (masinter) [inline lispusers/MODERNIZE:342]: is there an off-by-one here?  shouldn't LEFT > SCREENWIDTH be LEFT >= SCREENWIDTH (or SCREENWIDTH > LEFT)?

**Comment by rmkaplan** [comment]
I fixed the off-by-one bug, thanks.

**Review by masinter — APPROVED** [review]
i was running with this before and after the off-by-1 and the edits look reasonable and it worked fine. So...  LGTM

**Pull request merged**